src/main/ml-modules/root/com.marklogic.smart-mastering/matcher.xqy

```diff
@@ -130,6 +143,7 @@ declare function matcher:find-document-matches-by-options(
                               least this high to be returned
  : @param $lock-on-search  TODO
  : @param $include-matches  whether the response should list the matched properties for each potential match
+ : @param $filter-query  a cts:query used to filter the matched results
```

Comment:
"a cts:query used to restrict matches to a set, such as a specific entity type or collection"